src/extract.ts

```diff
@@ -270,18 +270,18 @@ async function getCommitFromGitHubAPIRequest(githubToken: string, ref?: string):
 
     return {
         author: {
-            name: commit.author.name,
-            username: data.author.login,
-            email: commit.author.email,
+            name: commit.author?.name,
+            username: data.author?.login,
+            email: commit.author?.email,
         },
         committer: {
-            name: commit.committer.name,
-            username: data.committer.login,
-            email: commit.committer.email,
+            name: commit.committer?.name,
+            username: data.committer?.login,
+            email: commit.committer?.email,
         },
         id: data.sha,
         message: commit.message,
-        timestamp: commit.author.date,
+        timestamp: commit.author?.date ?? commit.committer?.date,
```

Comment:
Any reason putting commiter date here when author date is null? This changes the existing behaviour without relating to this PR's purpose.

Reply:
That is true.
I fixed it not to change the behaviour c7c040c.
(The reason was that I thought it would be better to be able to get either timestamp)

Reply:
Thanks. I'm worried about the potential breaking change caused by the behaviour change. If you are still interested to make this change, feel free to have a seperated issue or PR to do the further investigation and evaluation, so it won't block the current PR.